UnityAds/src/main/java/com/applovin/mediation/adapters/UnityAdsMediationAdapter.java

```diff
@@ -73,13 +76,11 @@ public void initialize(final MaxAdapterInitializationParameters parameters, fina
             log( "Initializing UnityAds SDK with game id: " + gameId + "..." );
             initializationStatus = InitializationStatus.INITIALIZING;
 
-            if ( serverParameters.getBoolean( KEY_SET_MEDIATION_IDENTIFIER ) )
-            {
-                MediationMetaData mediationMetaData = new MediationMetaData( context );
-                mediationMetaData.setName( UnityAdsMediationAdapter.mediationTag() );
-                mediationMetaData.setVersion( AppLovinSdk.VERSION );
-                mediationMetaData.commit();
-            }
+            MediationMetaData mediationMetaData = new MediationMetaData( context );
+            mediationMetaData.setName( MEDIATION_TAG );
+            mediationMetaData.setVersion( UnityAds.getVersion() );
```

Comment:
Want to double-check if this is actually supposed to be the AppLovin MAX SDK version, instead of your SDK's version (I'd imagine your SDK would know its own SDK version).